deploy/030-node.yaml

```diff
@@ -29,7 +29,6 @@ spec:
           image: quay.io/kubevirt/csi-driver:latest
           args:
             - "--endpoint=unix:/csi/csi.sock"
-            - "--namespace=kubevirt-csi-driver"
```

Comment:
this arg is no longer used and caused the csi driver to fail since it's an unknown cli arg now